RetailCoder.VBE/Refactorings/EncapsulateField/EncapsulateFieldPresenterFactory.cs

```diff
@@ -7,10 +8,10 @@ namespace Rubberduck.Refactorings.EncapsulateField
     public class EncapsulateFieldPresenterFactory : IRefactoringPresenterFactory<EncapsulateFieldPresenter>
```

Comment:
is it ninject injecting a `IRefactoringPresenterFactory<EncapsulateFieldPresenter>`? If we find a way to bind that constructed generic interface `ToFactory`, then the concrete implementations can just go away.